modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -30,22 +30,24 @@ $ rosa create admin --cluster=<cluster_name>
 W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
 I: Admin account has been added to cluster 'cluster_name'. It may take up to a minute for the account to become active.
 I: To login, run the following command:
-oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \
+oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \ <1>
```

Comment:
Unless this is intentionally misspelled, I think there's a typo here:
```suggestion
oc login https://api.cluster-name.t6k4.i1.organization.org:6443 \ <1>
```

Reply:
Created a Jira: https://issues.redhat.com/browse/OSDOCS-9968 to implement out of scope comments for this PR. Thank you!